core/jobs/src/test/java/com/cognifide/aet/job/common/modifiers/hide/HideModifierTest.java

```diff
@@ -70,6 +72,32 @@ public void setParameters_XPathIsValid_ValidationPassedSuccessfuly() throws Para
     tested.setParameters(params);
   }
 
+  @Test
+  public void setParameters_CssIsValid_ValidationPassedSuccessfuly() throws ParametersException {
+    when(params.containsKey(PARAM_CSS)).thenReturn(true);
+    when(params.get(PARAM_CSS)).thenReturn(PARAM_CSS_VALUE);
+    when(properties.getUrl()).thenReturn(URL);
+    tested.setParameters(params);
+  }
+
+  @Test(expected = ParametersException.class)
+  public void setParameters_CssAndXpathArePassed_ValidationPassedUnsuccessfuly() throws ParametersException {
+    when(params.containsKey(PARAM_CSS)).thenReturn(true);
+    when(params.get(PARAM_CSS)).thenReturn(PARAM_CSS_VALUE);
+
+    when(params.containsKey(PARAM_XPATH)).thenReturn(true);
+    when(params.get(PARAM_XPATH)).thenReturn(PARAM_XPATH_VALUE);
+
+    when(properties.getUrl()).thenReturn(URL);
+    tested.setParameters(params);
+  }
+
+  @Test(expected = ParametersException.class)
+  public void setParameters_CssAndXpathAreNotPassed_ValidationPassedUnsuccessfuly() throws ParametersException {
+    when(properties.getUrl()).thenReturn(URL);
```

Comment:
It looks like this `properties` mock is ever used. If this is the case it shold be removed.
Then test methods from line 96 and 101 are the same? (must have)